### PR TITLE
Update SDK versions and switch Demo to use drawPath

### PR DESCRIPTION
**Pull request by christiefelker**
Bumps all demos (and documentation to v1.63.16)

Specifically changes the Demo to use `drawPath` instead of the navigator functionality

### CODE REVIEW

File: examples/Demo/index.js [hunk 1]
```diff
@@ -100,13 +100,40 @@ function getMapsInJourney(directions) {
...
 	}
 	var sortedMapIds = mapsSortedByElevation.filter(map => mapIds.indexOf(map.id) !== -1)
 	return sortedMapIds
 }
 
 
+function drawConnectionMarkers(directions, startPolygon, endPolygon) {
+	directions.instructions.forEach(function (instruction) {
+		if (instruction.action.type == "TakeVortex") {
+			var markerHTMLString = ''
```
Comment (seanclin) [inline examples/Demo/index.js:113]: Could initialize this as stairs.png and remove the last two conditions, but not a big deal

Comment (christiefelker) [reply examples/Demo/index.js:114]: yeah I like that better.

Comment (christiefelker) [reply examples/Demo/index.js]: Done @seanclin

**Comment by jessicarogobete** [comment]
while testing I noticed that the connection symbols are sometimes hard to see against the white background and the path interacts weirdly with the connection polygon
Screen Shot 2020-04-03 at 3 13 11 PM

Screen Shot 2020-04-03 at 4 05 10 PM

I don't know if this matters for this PR, but other than that it looks good. I tested random pathing both accessible and not accessible on the eaton centre.

**Comment by jessicarogobete** [comment]
re-tested the same things as before, looks good now. I did notice that the connection icons are a bit slow to respond to changes in zoom level (they catch up to where they should be a second after the zoom is done) but I'm not sure how much can be done to improve that.

**Pull request merged**